### DISCUSSION TITLE
PAPERLESS_CONSUMER_SUBDIRS_AS_TAGS not working?

### DISCUSSION THREAD

**Discussion by streiman** [jonaswinkler/paperless-ng#701] [Support]
I just installed paperless-ng and it looks great. I am inporting my digital archive and used the PAPERLESS_CONSUMER_SUBDIRS_AS_TAGS=true option.

Paperless-ng is creating the tags, but it is not assigning these tags to the consumed pdf. Am I missing something?

Thanks for any help
Andreas

**Comment by jonaswinkler** [comment]
I assume you also set PAPERLESS_CONSUMER_RECURSIVE=true?

**Reply by streiman** [reply]
Sorry, it was my fault. Working as expected, but takes time. Paperless-ng is creating the tags first, even if it is consuming the files in the subfolder later (when there are many files).
